unison-cli/src/Unison/CommandLine/InputPatterns.hs

```diff
@@ -1550,7 +1610,7 @@ help =
             intercalateMap
               "\n\n"
               showPatternHelp
-              (sortOn I.patternName validInputs)
+              visibleInputs
```

Comment:
The inputs are sorted in the original list, which means they'll appear in the expected order everywhere, and also means GHC will only sort the list the first time.